### PR TITLE
Update CONTRIBUTING.md for Grammar and Architecture Changes

### PR DESCRIPTION
**Pull request by TechyPirate**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,86 +1,87 @@
...
 
-F3D is separated in different components:
-- The F3D application, in the application folder.
-- The libf3d, in the library folder.
-- The VTKExtensions in the library/VTKExtensions folder.
-- The bindings, python, java and webassembly, in the respective directories.
+F3D is divided into different components:
+- The f3d executable, in the application folder.
+- The libf3d library, in the library folder.
+- The VTKExtensions, in the vtkext folder.
```
Comment (mwestphal) [inline CONTRIBUTING.md:74]: Extensions to vtk, in the vtkext folder.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,86 +1,87 @@
...
-VTKExtensions are separated in different modules.
-- Core, that do not depend on any other VTKExtensions modules are provide services for all modules
-- Readers, that depends on Core and implements many new VTK readers and importers
-- Rendering, that depends on Core and implements the rendering specificities of F3D
-- Applicative, the depends on all other VTKExtension modules and provide services for the libf3d library
+VTKExtensions are divided into different modules:
+- Core, that do not depend on any other VTKExtensions modules, provide services for all modules.
+- Readers, that depend on Core and implement many new VTK readers and importers.
+- Rendering, that depend on Core and implement the rendering specificities of F3D.
+- Applicative, that depend on all other VTKExtension modules and provide services for the libf3d library.
```
Comment (mwestphal) [inline CONTRIBUTING.md:82]: actually, this is not true anymore.

vtkext contains now only a public module that is documented in PLUGINS.md and a private module that is used exclusively by the libf3d.

**Review by mwestphal — CHANGES_REQUESTED** [review]
Small changes needed, but great work!

File: CONTRIBUTING.md (hunk 1 above)
Comment (TechyPirate) [reply CONTRIBUTING.md]: Updated in commit c1633c7.

File: CONTRIBUTING.md (hunk 2 above)
Comment (TechyPirate) [reply CONTRIBUTING.md]: I had a feeling this was no longer correct when I was not able to find directories or notations about certain modules being classified in the listed ways either in the source or Linux tarball, but wasn't sure, so I did not make a change here. Thanks for clarifying the current status and confirming my suspicions.

Updated the documentation as indicated via commit c1633c7.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,86 +1,85 @@
...
-F3D is separated in different components:
-- The F3D application, in the application folder.
-- The libf3d, in the library folder.
-- The VTKExtensions in the library/VTKExtensions folder.
-- The bindings, python, java and webassembly, in the respective directories.
+F3D is divided into different components:
+- The f3d executable, in the application folder.
+- The libf3d library, in the library folder.
+- The extensions to VTK, in the vtkext folder.
+- The bindings for python, java, and webassembly, in their respective directories.
```
Comment (snoyer) [inline CONTRIBUTING.md:75]:
```suggestion
- The `f3d` executable, in the `application` folder.
- The `libf3d` library, in the `library` folder.
- The extensions to VTK, in the `vtkext` folder.
- The bindings for Python, Java, and WebAssembly, in their respective folders.
```
I like backticks, do you guys like backticks too?

Comment (mwestphal) [reply CONTRIBUTING.md:75]: good point!

Comment (TechyPirate) [reply CONTRIBUTING.md:75]: Good catches on line 75!

I could go either way on the backticks really depending how I think about it.

Comment (mwestphal) [reply CONTRIBUTING.md:75]: lets go with backticks for directory names.

**Review by mwestphal — APPROVED** [review]

**Review by snoyer — APPROVED** [review]
more backticks while we're here, otherwise looks good

**Pull request merged**